src/main/java/com/minecolonies/coremod/colony/Structures.java

```diff
@@ -153,6 +148,50 @@ private static void loadStyleMaps()
         }
     }
 
+
+    private static void loadStyleMapsJar()
+    {
+        try
+        {
+            Log.getLogger().info("loadStyleMaps jar: " + ColonyManager.class.getResource(SCHEMATICS_ASSET_PATH).toURI()); 
+            final URI uri = ColonyManager.class.getResource(SCHEMATICS_ASSET_PATH).toURI();
+            if ("jar".equals(uri.getScheme()))
+            {
+                try (FileSystem fileSystem = FileSystems.getFileSystem(ColonyManager.class.getResource(SCHEMATICS_ASSET_PATH).toURI()))
+                {
+                    final Path basePath = fileSystem.getPath(SCHEMATICS_ASSET_PATH);
+                    Log.getLogger().info("Load huts or decorations from jar");
+                    loadSchematicsForPrefix(basePath, SCHEMATICS_PREFIX);
+                }
+                catch (@NotNull IOException | URISyntaxException | FileSystemNotFoundException e1)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1166)